<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyCSV</class>
 <widget class="QMainWindow" name="PyCSV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>251</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>251</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>251</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CsvCombine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWid">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>CsvCombine</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="inputPB">
              <property name="text">
               <string>Input Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputLE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="outputPB">
              <property name="text">
               <string>Output Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="outputLE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>218</width>
                <height>13</height>
               </size>
              </property>
              <property name="text">
               <string>New File Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameLE"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="combinePB">
              <property name="text">
               <string>Combine</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearPB">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>xls-&gt;csv</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="inputPB_2">
              <property name="text">
               <string>Input File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputLE_2">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="outputPB_2">
              <property name="text">
               <string>Output Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="outputLE_2">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>218</width>
                <height>13</height>
               </size>
              </property>
              <property name="text">
               <string>New File Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameLE_2"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="combinePB_2">
              <property name="text">
               <string>Convert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearPB_2">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHelp">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearPB</sender>
   <signal>clicked()</signal>
   <receiver>nameLE</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPB</sender>
   <signal>clicked()</signal>
   <receiver>outputLE</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPB</sender>
   <signal>clicked()</signal>
   <receiver>inputLE</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPB_2</sender>
   <signal>clicked()</signal>
   <receiver>nameLE_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPB_2</sender>
   <signal>clicked()</signal>
   <receiver>outputLE_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearPB_2</sender>
   <signal>clicked()</signal>
   <receiver>inputLE_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
